code/datums/mutations/_combined.dm

```diff
@@ -17,6 +17,10 @@
 	required = "/datum/mutation/human/strong; /datum/mutation/human/radioactive"
 	result = HULK
 
+/datum/generecipe/x_ray
+	required = "/datum/mutation/human/thermal; /datum/mutation/human/radioactive"
```

Comment:
using `" "` for typed paths is a sin.

Whoever designed this system to use this should be dragged out into the streets, promptly beaten, stabbed, and finally shot.

Reply:
jfc wow ok
It was originally a list, but they wanted me to do it without initializing it. You can’t use a datums list without Initializing the whole the thing

Reply:
If you got a better idea, tell me. Simply using a required_1 and required_2 would be a real maintainability hazard